frontend/src/employees/CreateEmployeeForm.js

```diff
@@ -30,17 +30,17 @@ const schema = yup.object().shape({
     .oneOf([yup.ref("password")], "A két jelszó nem egyezik meg!")
 });
 
-function CreateEmployeeForm({ sent, setSent, sentSuccessfully, popUpMessage, sendData, path, title }) {
+function CreateEmployeeForm({ sent, setSent, handleClick, popUpMessage, sendData, title }) {
+
   return (
     <div className="container my-5">
       <div className="row justify-content-center">
         <div className="col-12">
           {sent
             && <PopUp
+              handleClick={handleClick}
               body={popUpMessage}
-              sentSuccessfully={sentSuccessfully}
               setSent={setSent}
```

Comment:
Sajnálom, de szerintem a setSent is unused még.

Reply:
Jogos, töröltem. :)